bricksrc/quantities.py

```diff
@@ -1,5 +1,32 @@
+from brickschema.graph import Graph
+from brickschema.inference import BrickInferenceSession
 from rdflib import Literal
-from .namespaces import SKOS, OWL, BRICK
+from .namespaces import SKOS, OWL, RDFS, BRICK, QUDTQK, QUDT
+
+
+g = Graph()
+g.load_file("support/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl")
+g.load_file("support/VOCAB_QUDT-UNITS-ALL-v2.1.ttl")
+g.g.bind("qudt", QUDT)
+g.g.bind("qudtqk", QUDTQK)
+sess = BrickInferenceSession()
+g = sess.expand(g)
+
+
+def get_units(brick_quantity):
```

Comment:
It might be useful to include unit definitions too. I have no strong opinion on this, but Brick could become a single point for metadata schema with complete definitions.

On the other hand, I am not fully convinced to include the qudt symbols. It will be useful for some use cases, but including them inside Brick schema would confuse users IMHO.

Reply:
We can discuss this more on the call today: I think we want to minimize the amount of external information we include in the Brick definition that is replicated elsewhere -- it is easy for the number of definitions we include to grow and grow. If we start with QuantityKinds, then its Units, then its the dimensions and before we know it we have pulled in all of QUDT. This is not how the semweb world usually works -- it is more appropriate to pull in external graphs when they are needed. 

QUDT symbols are nice for knowing how to render a unit, but the symbols chosen by QUDT are not always the most obvious. We can explore including our own symbols or removing this altogether